doc/sdcard_burn.md

```diff
@@ -1,26 +1,34 @@
-# Writting the SD card image onto the SD card
+# Writing the SD card image onto the SD card
 
-The size SD card used to write the image to must be equal or greater than the size of the image.
+The size of the SD card that is going to be used to write the image to, must be equal or greater than the size of the image.
 Writing the image on the SD card will take about 15 to 20 minutes for a 16GB image, depending on the transfer speed of the system being used. 
 
 ## Linux
 * Make sure that xz-utils are installed on your machine
     ~~~sh
     sudo apt-get install xz-utils
     ~~~
-* Unpack the .tar.xz archive
+* Unpack the .xz archive
     ~~~sh
-    tar xf xxx.tar.gz
+    unxz name-of-the-image.xz
     ~~~
 * Write the image on the SD card
     ~~~sh
-    dd if=xxx.img of=/dev/xxx bs=4M
+    dd if=name-of-the-image.img of=/dev/xxx bs=4M
     sync
     ~~~
     
-Note: The **lsblk** command can be used to help identifying the device to be specified for the argument: **of=/dev/xxx**.
+Note 1: The following command can be used to help identifying the device than needs to be specified for the argument **of=/dev/xxx**:
```

Comment:
I think here should be **that**: "....the device than needs to be specified...."

Reply:
yes, fixed. Thanks!